Libraries/StyleSheet/StyleSheetTypes.js

```diff
@@ -637,11 +637,11 @@ export type ____TextStyle_Internal = $ReadOnly<{|
     | 'tabular-nums'
     | 'proportional-nums',
   >,
-  apple_fontSmoothing?:
+  apple_fontSmoothing?:  // [macOS
     | 'auto'
     | 'none'
     | 'antialiased'
-    | 'subpixel-antialiased', // TODO(OSS Candidate ISS#2710739)
+    | 'subpixel-antialiased', // macOS]
```

Comment:
Just wanted to check, we're sure it's not just line 644 that needs the tag? The whole block needs it?

Reply:
Yeah, the whole prop and type is an rn-macOS addition.